target/linux/mediatek/image/filogic.mk

```
@@ -583,6 +583,28 @@ endif
 endef
 TARGET_DEVICES += bananapi_bpi-r4-poe
 
+define Device/buffalo_wsr-6000ax8-common
+  DEVICE_VENDOR := Buffalo
+  DEVICE_DTS_DIR := ../dts
+  DEVICE_PACKAGES := kmod-mt7915e kmod-mt7986-firmware mt7986-wo-firmware
+  IMAGE_SIZE := 26624k
+  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
+endef
+
+define Device/buffalo_wsr-6000ax8
+  DEVICE_MODEL := WSR-6000AX8
+  DEVICE_DTS := mt7986b-buffalo-wsr-6000ax8
+  $(call Device/buffalo_wsr-6000ax8-common)
+endef
+TARGET_DEVICES += buffalo_wsr-6000ax8
+
+define Device/buffalo_wsr-6000ax8p
+  DEVICE_MODEL := WSR-6000AX8P
+  DEVICE_DTS := mt7986b-buffalo-wsr-6000ax8p
+  $(call Device/buffalo_wsr-6000ax8-common)
+endef
+TARGET_DEVICES += buffalo_wsr-6000ax8p
+
```

Do you need two different images?

I would prefer only one image:
```suggestion
define Device/buffalo_wsr-6000ax8
  DEVICE_MODEL := WSR-6000AX8
  DEVICE_VENDOR := Buffalo
  DEVICE_ALT0_MODEL := WSR-6000AX8P
  DEVICE_ALT0_VENDOR := Buffalo
  DEVICE_DTS_DIR := ../dts
  DEVICE_DTS := mt7986b-buffalo-wsr-6000ax8
  DEVICE_PACKAGES := kmod-mt7915e kmod-mt7986-firmware mt7986-wo-firmware
  IMAGE_SIZE := 26624k
  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
endef
TARGET_DEVICES += buffalo_wsr-6000ax8

```